packages/interface/src/contexts/Round.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { createContext, useContext, useMemo } from "react";
+
+import type { RoundContextType, RoundProviderProps } from "./types";
+import type { Round } from "~/features/rounds/types";
+
+export const RoundContext = createContext<RoundContextType | undefined>(undefined);
+
+export const RoundProvider: React.FC<RoundProviderProps> = ({ children }: RoundProviderProps) => {
+  const rounds = [
+    {
+      roundId: "open-rpgf-1",
+      description: "This is the description of this round, please add your own description.",
+      startsAt: 1723477832000,
+      registrationEndsAt: 1723487832000,
+      votingEndsAt: 1724009826000,
+      tallyURL: "https://upblxu2duoxmkobt.public.blob.vercel-storage.com/tally.json",
+    },
```

Comment:
What's the reason to keep these config fields here?

Reply:
just as I mentioned in the description, these data should be got from the registry contract, now I'm testing out the frontend components, so just put fake data here.

Reply:
This data will be available using metadataUrl from registry contract. Registry won't have this data.

Reply:
yes but still need these fake data to test the frontend? or you suggest to upload these data as json file to vercel and add the URL in the env variable?

Reply:
yes sure, it should be a service to fetch these metadata from the metadataURL provided by registry contract, and the registry contract address is provided inside the env file.
The metadataURL is added once the user start a new round --> so once a user setup a new round and click deploy, the data are uploaded to a certain storage, and a url is added to the registry contract.

I think this is the full process of the round information, and it shouldn't be included in this PR, that's why I use fake data now.

Reply:
Sure, for now let's keep fake data.